### ISSUE TITLE
Arc doesn't render to pyglet.image.Framebuffer

### ISSUE THREAD

**Issue by kingadami** [pyglet/pyglet#1023]
Basically rendering shapes to a texture works for all shapes except for the pyglet.shapes.Arc class.  I believe this is due to it having its own draw method and therefore the opengl state isn't setup correctly.  Removing Arc's draw method and letting it use the one in ShapeBase appears to fix the issue.

Program used to show the problem and verify it is fixed:
```
#!/usr/bin/env python3

import pyglet
import math

class Board(pyglet.window.Window):
    def __init__(self):
        super().__init__(caption="Test")

        self.image = pyglet.image.create(self.size[0],self.size[1])
        self.buffer = pyglet.image.Framebuffer()
        self.buffer.attach_texture(self.image.get_texture())

        label = pyglet.text.Label("Hello World", font_name="Times New Roman", font_size=36, x=self.size[0]//2, y=self.size[1]//2, anchor_x="center", anchor_y="center")

        # Draw to our image
        self.buffer.bind()
        label.draw()
        pyglet.shapes.Line(0,0,self.image.width,self.image.height,10,(255,0,0)).draw()
        pyglet.shapes.Line(0,self.image.height,self.image.width,0,10,(255,0,0)).draw()
        pyglet.shapes.Arc(self.image.width / 2, self.image.height / 2,min(self.image.width / 4, self.image.height / 4), color=(0,0,255)).draw()
        self.buffer.unbind()

        self.batch = pyglet.graphics.Batch()
        self.sprite = pyglet.sprite.Sprite(img=self.image, batch=self.batch)
        self.arc = pyglet.shapes.Arc(self.image.width / 2, self.image.height / 2,min(self.image.width / 6, self.image.height / 6), color=(255,0,255), batch=self.batch)

    def on_draw(self):
        self.clear()
        self.batch.draw()

if __name__ == "__main__":
    board = Board()
    pyglet.app.run()
```

This program should display 2 arcs. a blue one and a smaller pink one.  The blue one will not draw with the Arc's draw method present but the pink one will.  If you remove Arc's draw method then both are visible.

This appears to be present on at least the master branch commit [8bef88a](https://github.com/pyglet/pyglet/commit/8bef88a5922c13cd5ff466f300a2610ce43eacbb)

**Comment by benmoran56** [comment]
Looks like that overridden draw method was left in there by accident. I've removed it in master, and will be in the next point release. 

Also, you may already be aware, but using the `draw` methods on individual pyglet objects is not recommended. It forces a complete setup/teardown of GL state & an individual OpenGL draw for each time. Batches should always be used for performance.

**Comment by kingadami** [comment]
Thanks.  I do know batches are more efficient and was using the direct draw methods for the rendering to texture out of laziness because it only happens once.  Although, now I know that the things added to a Batch instance don't have their own draw methods called.  It makes sense when looking at it all because the Batch collects all of the vertex lists and draws them directly.  At least I found code which shouldn't be there anymore for you to remove.

I am porting a simple Tic Tac Toe game I made a long time ago from pygame to pyglet in order to try to learn pyglet.  So I am sure I will find more things because I will probably be using it in not as common ways.
